_posts/2017/2017-02-20-creating-a-sales-analytics-app-with-ruby-on-rails.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: "Creating a Sales Analytics App with Ruby on Rails"
+date: "2017-02-20 07:00:00"
```

Comment:
We can talk about a publication date once the review is done. Depending on when you can make it, I suggest either the 21st or 23rd of February.